packages/contentstack-audit/README.md

```diff
@@ -19,7 +19,7 @@ $ npm install -g @contentstack/cli-audit
 $ csdx COMMAND
 running command...
 $ csdx (--version|-v)
-@contentstack/cli-audit/1.13.0 darwin-x64 node-v22.17.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.